atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -223,7 +233,9 @@ def run(self):
         if pulp_push_results:
             media_types += ['application/json']
 
-        pulp_pull_results = self.workflow.postbuild_results.get(PulpPullPlugin.key)
+        # pulp_pull may run on worker as a postbuild plugin or on orchestrator as an exit plugin
+        pulp_pull_results = self.workflow.postbuild_results.get(PulpPullPlugin.key) or \
+            self.workflow.exit_results.get(PulpPullPlugin.key)
```

Comment:
Nit: nicer to use parentheses and line these up, instead of backslash

Reply:
Good idea, implemented